crates/engine/src/mock_rocksdb_engine.rs

```diff
@@ -9,12 +8,12 @@ use bytes::{Bytes, BytesMut};
 use crate::{
     api::{engine_api::StorageEngine, snapshot_api::SnapshotApi},
     error::EngineError,
-    memory_engine::{MemoryEngine, MemorySnapshot},
-    TransactionApi, WriteOperation,
+    memory_engine::{MemoryEngine, MemorySnapshot, MemoryTransaction},
+    StorageOps, TransactionApi, WriteOperation,
 };
 
 /// Mock `RocksDB` Storage Engine
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Same as above. Why shoule a `RocksEngine` be cloneable?

Reply:
It's a mock implementation, and it's a wrapper around a `MemoryEngine`, which is clonable.